### ISSUE TITLE
NoGUI executales are blocked for download by browsers due to malicious content.

### ISSUE THREAD

**Issue by sergetensen** [benpye/wsl-ssh-pageant#50]
I tried to download but both 32- and 64 no-gui versions are reported by Chrome, FireFox and Edge as risk files and blocked by default.
Virustotal shows many av-vendors reporting trojan like code in there. I'm not a coder but if I can do anything to help to make this software work safely please let me know.

//Serge

**Comment by benpye** [comment]
Hey Serge,

This appears to be a duplicate of #38 . Unfortunately AV software does a pretty bad job with binaries written with Go and often result in false detections. I would suggest if you are concerned building the project locally, it should be fairly straightforward.

1. Install Go from https://golang.org/dl/
2. Get a copy of the code - if you don't have git I think you can use the zip from https://github.com/benpye/wsl-ssh-pageant/archive/refs/heads/golang.zip
3. Navigate to the source directory in PowerShell and run `build.ps1`

I'm going to close this issue in favour of #38
